packages/EasyStandard/docs/services/rector.md

```diff
@@ -0,0 +1,180 @@
+---eonx_docs---
+title: 'EasyStandard: Rector'
```

Comment:
Let's change `Rector` to `Rectors` and rename the file accordingly.

Reply:
Done.